tests/specs/mcd/vat-move-diff-rough-spec.k

```diff
@@ -76,7 +76,16 @@ module VAT-MOVE-DIFF-ROUGH-SPEC
               <acctID> ACCT_ID </acctID>
               <balance> ACCT_ID_balance </balance>
               <code> Vat_bin_runtime </code>
-              <storage> ACCT_ID_STORAGE => ACCT_ID_STORAGE [ #Vat.dai[ABI_src] <- Dai_src -Int ABI_rad ] [ #Vat.dai[ABI_dst] <- Dai_dst +Int ABI_rad ] </storage>
+              <storage>
+                (#Vat.can[ABI_src][CALLER_ID] |-> May)
+                (#Vat.dai[ABI_src] |-> Dai_src)
+                (#Vat.dai[ABI_dst] |-> Dai_dst)
+                ACCT_ID_STORAGE =>
+                  (#Vat.can[ABI_src][CALLER_ID] |-> May)
+                  (#Vat.dai[ABI_src] |-> Dai_src -Int ABI_rad)
+                  (#Vat.dai[ABI_dst] |-> Dai_dst +Int ABI_rad)
+                  ACCT_ID_STORAGE
+              </storage>
```

Comment:
This implicitely assumes that `#Vat.dai[ABI_src]` and `#Vat.dai[ABI_dst]` don't alias, which is true for this test, but not present here is teh `vat-move-same-rough-spec.k` test, which _does_ assume they alias. So just wondering if we need to have this change, because the previous way of expressing it didn't lead to having any questions about the aliasing.

Reply:
Oh that's interesting, yes, they could be the same.